ecs-agent/acs/session/session.go

```diff
@@ -278,7 +278,7 @@ func (s *session) startACSSession(ctx context.Context, client wsclient.ClientSer
 		client.AddRequestHandler(r.HandlerFunc())
 	}
 
-	if s.dockerVersion == "containerd" && s.resourceHandler != nil {
```

Comment:
Sry may miss some context here. Why are we removing this check

Reply:
Good question. We're going to be using the existing resource responder for docker agent as well.